odk1-src/form-question-types.rst

```diff
@@ -1043,8 +1043,18 @@ To include images as choices for select questions:
      make sure the media folder is placed in
      :file:`/sdcard/odk/forms/`.
 
-  
+.. _or-other:  
+
+Including "other" as a choice
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+On the **survey** sheet, in the :th:`type` column,
+after the type and the list_name,
+you can add :tc:`or_other`.
+This will add "Other" as an additional option to your choice list.
```

Comment:
Did you intentionally leave option here? It's not bad (maybe a wee bit 😂), I'm just curious.

Reply:
I did! (I almost made a note about it, too.)